server/index.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const express = require('express');
+const models = require('./models/api.js');
+
+const PORT = 2373;
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
```

Comment:
Do you need express.json()? I believe it's just used to parse JSON data in the request, which you won't have because you're only accepting GET requests.

Reply:
Great point! It's become a habit 😆